index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  
+
+  <section id="headline">
+    <header>
+      <a href="#" id="logo">LOGO</a>
+        <div id="mobile-navbar">
+          <div id="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </div>
+        <nav id="nav-container">
+          <ul id="nav-menu">
+            <li class="nav-link"><a href="#works">Portfolio</a></li>
+            <li class="nav-link"><a href="#about-myself">Myself</a></li>
+            <li class="nav-link"><a href="#form-section">Contact</a></li>
+          </ul>
+      </nav>
+    </header>
+    <div id="headline-content">
+      <h1 class="header"> <br>    <br> Tanusri from India <br> Have a good day!</h1>
+      <p class="portfolio-description">I am a web developer. Need Web Development ----- Get in touch with me
+      </p>
+      <h2 class="lets-connect">CONNECT ME</h2>
+      <ul class="social-media-links">
+        <li> <a href="https://www.linkedin.com/in/tanusri-ghosh-2a56b814"> LinkedIn </a> </li>
+	      <li> <a href="https://twitter.com/chuaghosh25"> Twitter </a> </li>
+	      <li> <a href="https://www.google.com/tghosh25@gmail.com"> google </a> </li>
+      </ul>
+    </div>
+  </section>
```

Comment:
- [ ] As requested by the previous reviewer Kindly fix the layout of your Headline section, it's barely visible. Be sure that the background is also visible as per [FIGMA design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
### Your design
![hiss](https://user-images.githubusercontent.com/81727988/186216112-997dd22a-9980-4aae-97ac-0af97d253d3a.png)
### Figma design
![figma design](https://user-images.githubusercontent.com/81727988/186216186-a923fe4e-bcf5-4bec-88c9-11f6ddd13d91.png)